andela/hc-aces-kla
# Ft priority team members #158174599

## Status
**READY**

## Migrations
YES

## Description
User should be able to set up a priority list for users to be notified one after the other after a job goes down.

## Related PRs
None

## Todos
- [x] Views
- [x] Profile UI


## Deploy Notes
On deployment run migrations ```./manage.py migrate```

Run background processes  ```./manage.py ensuretriggers``` and ```./manage.py sendalerts```

## Steps to Test or Reproduce

```sh
* Clone the repo, run migrations. First run ```./ manage.py makemogrations``` then ```./ manage.py migrate```
* Run the application ```./ manage.py runserver```.
* Open the app in the browser, log in and invite team members.
* Go to the account settings and invite team members.
* Assign the different users checks and priority.
* If any of the checks assigned to the team members, they should be notified by the prioirity given
```

## Impacted Areas in Application

Edited
* hc/accounts/views.py - Added the assign_checks that handles the logic for assigning and un assigning checks
* hc/templates/accounts/profile.html - Added view assigns link and assign check modal.
* hc/front/tests/test_add_profile.py - Added tests for the assigning checks
* hc/static/js/profile.js - Added jquery to handle the check boxes.

##Screen shots
<img width="601" alt="screen shot 2018-07-23 at 17 22 49" src="https://user-images.githubusercontent.com/39958516/43082593-162e0528-8e9d-11e8-9e37-618b51a6dee2.png">
<img width="647" alt="screen shot 2018-07-23 at 17 22 41" src="https://user-images.githubusercontent.com/39958516/43082614-22482424-8e9d-11e8-9f3b-42f08d4189d1.png">
